dsc/tests/dsc_export.tests.ps1

```diff
@@ -0,0 +1,45 @@
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT License.
+
+Describe 'resource export tests' {
+    
+    It 'Export can be called on individual resource' {
+
+        $processes = dsc resource export -r Microsoft/Process
+        $LASTEXITCODE | Should -Be 0
+        $processes.count | Should -BeGreaterThan 1
+    }
+
+    It 'Export can be called on a configuration' {
+
+        $json = @'
+            $schema: https://schemas.microsoft.com/dsc/2023/03/configuration.schema.json
+            resources:
+            - name: Processes
+              type: Microsoft/Process
+              properties:
+                pid: 0
+'@
+        $out = $json | dsc config export
+        $LASTEXITCODE | Should -Be 0
+        $config_with_process_list = $out | ConvertFrom-Json
+        $config_with_process_list.resources.count | Should -BeGreaterThan 1
```

Comment:
Not a blocker for this PR, but we should probably have an env var test resource instead of process since it's something we can more easily control and validate

Reply:
I agree;  env var test resource would be better here.